Butterfly.Core/Database/BaseDatabase.cs

```diff
@@ -385,7 +385,8 @@ public abstract class BaseDatabase : IDatabase {
             else if (typeText.Equals("DOUBLE", StringComparison.OrdinalIgnoreCase)) {
                 type = typeof(double);
             }
```

Comment:
This function's name (`ConvertMySqlType`) should probably change as it's used by the `CreateStatement` for all providers. Alternatively it might be removed completely if/when the `*Statement`s get their own provider implementations.

Reply:
Unfortunately, our CREATE statement syntax is MySQL syntax specific.  This isn't great as someone familiar with Postgres or SQL Server would not want to use MySQL syntax.  

The only exception to this is the MemoryDatabase implementation which doesn't really have a native CREATE syntax at all (it's just using an in-memory DataTable).

Perhaps we change our design a bit to just allow executing "native SQL" commands that are not parsed by us and we use these methods to execute things that don't generate data change events (like CREATE statements).  With this change, the CREATE statements would be created in the native syntax of the database being used.  We might continue to use the current MySQL syntax for the MemoryDatabase as it needs some way to create tables. This all seems beyond the scope of the Postgres implementation.

Reply:
Agreed yes that should work fine. For now, this change works with Postgres.